layouts/partials/navigation.html

```diff
@@ -1,95 +1,131 @@
-<nav class="navbar navbar-expand-md navbar-dark d-flex flex-column p-0" style=" {{ if .IsHome }}background-color: transparent{{ else }}background-color: {{ site.Params.primary_color }}{{ end }}">
-  <div class="w-100 container mx-0 navigation-bar">
-    <div id="site-brand">
-      {{ $logo:= site.Params.logo }}
-      {{ $orgName:= site.Params.brand.parent_org_name }}
-      {{ $orgUrl:= site.Params.brand.parent_org_url }}
+<div>
```

Comment:
Was there a reason to add a `<div>` tag outside of the the `<nav>` section? This div wraps around the navigation element which feels awkward.